ci/vars-functional-test.yml

```diff
@@ -0,0 +1,7 @@
+---
+# temp: skip os-net-setup
+cifmw_os_net_setup_config: []
```

Comment:
I think skip os-net is no longer needed as we use post_deploy hook

Reply:
I want to test that separately, just to make sure.